tests/unit/src/test/scala/tests/rangeFormatting/MultilineStringRangeFormattingWhenPastingSuite.scala

```diff
@@ -319,10 +327,9 @@ class RangeFormattingWhenPastingSuite extends BaseLspSuite("rangeFormatting") {
        |object Main {
        |  val str = s'''
        |               |ok'''.stripMargin
-       |$indent
        |  val other = '''
-       |              |  some text
-       |              |'''.stripMargin
+       |  |  some text
```

Comment:
The indentation should not have changed here I think

Reply:
This case is not handled by the MultilineStringFormatter, or at least it seems it's not, so the `rangeFormatter` falls back on the `IndentOnPaste` formatter that tries to indent it as normal code, and not multiline strings.

Reply:
Hmm... but if the code is already indented in the paste snippet, shouldn't we keep the indentation? The same as with any block that has parts of it indented.

Reply:
I haven't actually checked the code but if the `MultilineString` formatter doesn't return an empty list, but a `None`, the formatting "passes" to the `IndentOnPaste` that doesn't consider "correct" the multiline-string formatting, or well, in this particular cases behave like this:

1. finds the correct indentation for the first line
2. finds the correct indentation for the second line based on the first one
3. computes all other lines according to the difference between the pasted indentation and the second line

It works like this because, as already discussed, the first line can either be pasted from the beginning of the line having references the copied indentation, or be pasted already in the correct indentation level not having those references. So the computation for the difference in indentation between the pasted lines and the current file is done comparing the second line to the others.

Reply:
However, in this case ` val` is indented. We could tweak the algorithm in this case to adjust the intentation on the next lines starting with the second:
```
newIndentation = secondLineIndentation - firstLineIndentation + caluclatedIndetationForFirstline
```
Otherwise, if the second line is indented, we will always get it wrong.

Or, we could special case `|` and other operators to be always indented, which would work in most cases I think